models/mailchimp/mailchimp_email_summary.sql

```diff
@@ -35,11 +35,11 @@ unsubscribes as (
 select
     s.campaign_id, s.email_id, sent_date, hard_bounced_date, first_opened_date,
     last_opened_date, total_opens, first_clicked_date, last_clicked_date,
-    total_clicks, unsubscribed_date--,
-    --decode(bounced_date, null, 0, 1) as hard_bounced,
-    --decode(opened_date, null, 0, 1) as opened,
-    --decode(clicked_date, null, 0, 1) as clicked,
-    --decode(unsubscribed_date, null, 0, 1) as unsubscribed
+    total_clicks, unsubscribed_date,
+    decode(hard_bounced_date, null, 0, 1) as hard_bounced,
+    decode(first_opened_date, null, 0, 1) as opened,
+    decode(first_clicked_date, null, 0, 1) as clicked,
+    decode(unsubscribed_date, null, 0, 1) as unsubscribed
```

Comment:
Yes but I used these in multiple subsequent analyses; I would prefer not to have to recalculate them repeatedly in every query. It's a really important fact about every email sent. And Redshift won't actually perform the aggregation if it's not needed to deliver the final results.